src/ucs/sys/preprocessor_external.h

```diff
@@ -0,0 +1,29 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+
+#ifndef UCS_PREPROCESSOR_EXTERNAL_H
+#define UCS_PREPROCESSOR_EXTERNAL_H
+
+/* Packed structure */
+#define UCS_S_PACKED __attribute__((packed))
```

Comment:
pls indent the macro "values" on column (i know indentation was not correct in original file as well, but we may use this opportunity to fix)